EpikCord/__init__.py

```diff
@@ -2279,6 +2280,23 @@ def latency(self):
     def average_latency(self):
         return sum(self.latencies) / len(self.latencies)
 
+    def add_check(self, check:Callable, interval:Optional[int]=None, **kwargs):
+        async def full_check(check, interval=5, **kwargs):
+            task_start = False
+            if kwargs["start"] and kwargs["until"]:# Start when and do until
+                while True:
+                    if time() == kwargs["start"] or task_start == True:
+                        await check()
+                        task_start = True
+                        await asyncio.sleep(interval)
+
```

Comment:
This looks inefficient way to do tasks, couldn't you think of something else ?

Reply:
pls suggest another way if applicable

Reply:
Is this tasks or checks ?

Reply:
tasks... look at description i made a mistake

Reply:
I don't know how to explain, but why not something like `@task(hour=1, minute=2)` ?, Then doing forever loop till cancel or the task errors, sleeping after each loop, and a way to tell max loops

Reply:
i did 3 of what you told

Reply:
i must commit that lol

Reply:
What about remaining 3 ?

Reply:
Anyways, i don't think using time() for starting will work, i don't know how to explain that part.

Reply:
Or better yet, rethink about ways of doing this, take a look at dpy's way of doing it.

Reply:
makes sense , will add the part

Reply:
Maybe doing a better way such as
```py
async def task(self, *, seconds, minutes, hours, count):
    async def inner(func):
        time = calculate_time(seconds, minutes, hours)
        callback = None
        if count:
            async def callback():
                for i in range(count):
                    await func()
                    await sleep(time)
        else:
            async def callback():
                await func()
                await sleep(time)
                await asyncio.call_later(time, callback)
        asyncio.get_event_loop().run_until_complete()
```